### ISSUE TITLE
Circular contigs in a Eukaryotic assembly

### ISSUE THREAD

**Issue by artorias111** [chhylp123/hifiasm#370]
When I assemble PacBio HiFi reads of a Eukaryotic linear genome through HiFiasm, I get a couple of circular contigs. The linear contigs are appended with an `l` in the fasta header, and the circular contigs are appended with a `c` in the fasta header. None of the circular genomes consist of mitochondrial DNA. 
On what basis is `l` and `c` assigned to these contigs? Thanks!

**Comment by chhylp123** [comment]
Yes, `l` is appended for linear contigs and `c` is appended for the circle contigs.
